mesonbuild/dependencies/pkgconfig.py

```diff
@@ -125,8 +204,34 @@ def _call_pkgbin_real(self, args: T.List[str], env: T.Dict[str, str]) -> T.Tuple
         return p.returncode, out.strip(), err.strip()
 
     @staticmethod
-    def get_env(environment: 'Environment', for_machine: MachineChoice,
-                uninstalled: bool = False) -> 'EnvironmentVariables':
+    def check_pkgconfig(env: Environment, pkgbin: ExternalProgram) -> T.Optional[str]:
+        if not pkgbin.found():
+            mlog.log(f'Did not find pkg-config by name {pkgbin.name!r}')
+            return None
+        command_as_string = ' '.join(pkgbin.get_command())
+        try:
+            helptext = Popen_safe(pkgbin.get_command() + ['--help'])[1]
+            if 'Pure-Perl' in helptext:
+                mlog.log(f'found pkg-config {command_as_string!r} but it is Strawberry Perl and thus broken. Ignoring...')
```

Comment:
I don't think it should be capitalized, elsewhere it has a capital because it's the first word of the sentence. Fixed another place I found it capitalized when not first word of sentence.

Reply:
Oh is this not the first word of the sentence?

Reply:
"found" is the first word, capitalized it now. Every small details matters I guess... :D

Reply:
Sorry, I misread what you said, I thought you meant the "P" in "pkg-config" lol